application/src/main/java/eu/europa/ec/fisheries/uvms/activity/rest/resources/ReportDocumentResource.java

```
@@ -43,10 +45,23 @@ This file is part of the Integrated Fisheries Data Management (IFDM) Suite. The
     @Produces(value = {MediaType.APPLICATION_XML})
     @Consumes(value = {MediaType.APPLICATION_XML})
     public Object getReport(FLUXFAQueryMessage fluxfaQueryMessage) throws JAXBException {
-        SubscriptionDataRequest subscriptionDataRequest = SubscriptionMapper.mapToSubscriptionDataRequest(fluxfaQueryMessage.getFAQuery());
-        List<SubscriptionDataCriteria> queryCriteria = subscriptionDataRequest.getQuery().getCriteria();
-        log.debug(queryCriteria.toString());
-        FLUXFAReportMessage reportsByCriteria = faQueryService.getReportsByCriteria(queryCriteria);
+        DateTimeType startDate = fluxfaQueryMessage.getFAQuery().getSpecifiedDelimitedPeriod().getStartDateTime();
```

Maybe some input checks so we can return a nice error message if the query is missing or has no specifiedDelimitedPeriod?

---

That is already handled by the API, if none is specified then search retrieves everything